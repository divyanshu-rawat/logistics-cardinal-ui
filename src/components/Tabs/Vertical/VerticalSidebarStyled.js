import styled from 'styled-components';

const VerticalSidebarStyled = styled('div')`
  background-color: ${({ theme }) => theme.colors.grey900};
  min-width: 220px;
`;

export default VerticalSidebarStyled;
