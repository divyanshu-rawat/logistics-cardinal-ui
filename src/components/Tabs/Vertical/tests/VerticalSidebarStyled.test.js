import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import VerticalSidebarStyled from '../VerticalSidebarStyled';
import Theme from '../../../../themes/rooster';

const renderComponent = () => <VerticalSidebarStyled theme={Theme} />;

describe('<VerticalSidebarStyled />', () => {
  it('should render the component', () => {
    const tree = toJson(shallow(renderComponent()));
    expect(tree).toMatchSnapshot();
  });
});
