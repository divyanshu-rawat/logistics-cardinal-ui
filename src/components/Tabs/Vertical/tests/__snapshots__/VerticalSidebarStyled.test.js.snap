// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VerticalSidebarStyled /> should render the component 1`] = `
<div
  className="sc-bdVaJa bFMOCn"
/>
`;
